tests/coord.py

```diff
@@ -0,0 +1,791 @@
+#!/usr/bin/env python2
```

Comment:
It's not possible to work out what had to change in the tests as the file is entirely new.